File: app/src/main/java/com/neome/java/api/meta/base/dto/FieldSetOfDocument.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnDocFileExt;
import com.neome.java.api.meta.base.Types.MediaIdDocument;

@SuppressWarnings("unused")
public class FieldSetOfDocument
{
  public EnumDefnDocFileExt[] fileExtSet;

  public String[] fileNameSet;

  public long[] fileSizeSet;

  public MediaIdDocument[] mediaIdDocumentSet;
}

File: app/src/main/java/com/neome/api/meta/base/dto/FieldSetOfDocument.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnDocFileExt
import com.neome.api.meta.base.Types.MediaIdDocument

interface FieldSetOfDocument
{
  val fileExtSet: Array<EnumDefnDocFileExt>
  val fileNameSet: Array<String>
  val fileSizeSet: Array<Number>
  val mediaIdDocumentSet: Array<MediaIdDocument>
}